packages/create-astro/src/actions/dependencies.ts

```diff
@@ -26,15 +26,6 @@ export async function dependencies(
 			start: `Dependencies installing with ${ctx.pkgManager}...`,
 			end: 'Dependencies installed',
 			while: () => {
-				return Promise.reject('Unknown error').catch((e) => {
```

Comment:
Prematurely returning from the function here caused that the install step would never run